scripts/sanity_runnable.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -eu
+
+TEST_PROJECT="smoketest"
+
+trap 'catch $? $LINENO' EXIT
+
+catch() {
+  if [[ "$1" != "0" ]]; then
+    echo "Error $1 occurred on $2"
+  fi
+
+  # return to origin, clear directory stack
+  pushd -0 > /dev/null && dirs -c
```

Comment:
TIL about `pushd` and `popd`! https://opensource.com/article/19/8/navigating-bash-shell-pushd-popd

Thank you! 🙂

Reply:
one of the most useful bash builtins